docs/Examples/get-tx-from-chain.js

```diff
@@ -1,46 +1,70 @@
 'use strict';
 
 const bcoin = require('../..');
-const Logger = require('blgr');
 
-// Setup logger to see what's Bcoin doing.
-const logger = new Logger({
-  level: 'debug'
-});
-
-// Create chain for testnet, specify chain directory
+// Create chain for testnet, stored in memory by default
 const chain = new bcoin.Chain({
-  logger: logger,
   network: 'testnet',
-  db: 'leveldb',
-  prefix: '/tmp/bcon-testnet',
```

Comment:
Having prefix may be useful to have in the example, as that usage is far more common and necessary.